config/clients/go/template/client/client.mustache

```diff
@@ -2182,6 +2224,9 @@ func (client *{{appShortName}}Client) WriteAssertionsExecute(request SdkClientWr
 	if authorizationModelId == nil {
 		return nil, FgaRequiredParamError{param: "AuthorizationModelId"}
 	}
+	if !utils.IsWellFormedUlidString(*authorizationModelId) {
```

Comment:
Can we move this inside the `client.getAuthorizationModelId` method so that we don't have to repeat it like in the sections above? If the auth model id is not nil, validate it

Reply:
done